dotCMS/src/main/java/com/dotmarketing/portlets/browser/ajax/BrowserAjax.java

```diff
@@ -2245,4 +2245,8 @@ public boolean deleteHTMLPagePreCheck(String htmlPageInode) throws Exception{
     	}
 	}
 
+	public String getActiveFolderInode(){
+		return activeFolderInode;
```

Comment:
I am not pretty sure storing attributes on share components is a good practice since multiple threads can hit or change this value, however if you need to have a share variable you must mark it as a transient

Reply:
I'm not sure it's completely true https://stackoverflow.com/questions/3544919/what-are-transient-and-volatile-modifiers